test/jdk/java/awt/Headless/HeadlessMalfunctionTest.java

```diff
@@ -38,7 +38,7 @@
  * @run driver  jdk.test.lib.helpers.ClassFileInstaller
  *              HeadlessMalfunctionAgent
  *              HeadlessMalfunctionAgent$1
- * @run driver HeadlessMalfunctionTest
+ * @run driver/timeout=240 HeadlessMalfunctionTest
```

Comment:
On my system this runs in about 2.5 seconds.
Our CI system  -which runs jtreg with concrurrency=4 - on a Linux VM reports a total of 12 seconds (including compilation) and the actual test execution is about 6 seconds.

That was x64 - a Linux ARM system was a total of 10 seconds.

So if you need 240 seconds, then I think you need to look into why it is > 10x slower than I see.


#section:driver
----------messages:(8/309)----------
command: driver HeadlessMalfunctionTest
reason: User specified action: run driver HeadlessMalfunctionTest 
started: Wed Sep 24 19:20:07 UTC 2025
Mode: othervm
Additional options from @modules: --add-modules java.desktop
Process id: 2328064
finished: Wed Sep 24 19:20:13 UTC 2025
elapsed time (seconds): 6.077

Reply:
What's the command you are using to run the tests ?
make test .... something .. ?

Reply:
I think that given we need stability in tests, the concurrency of 48 on a 48 CPU system may be too much.
Some tests might themselves need multiple threads.

I think the build system should reduce it to maybe 75% of what it uses for the build.
But that's a discussion for a different forum.